test/e2e/deployment/path_condition_match_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build e2e
+// +build e2e
+
+package deployment
+
+import (
+	. "github.com/onsi/ginkgo"
+	contourv1 "github.com/projectcontour/contour/apis/projectcontour/v1"
+	"github.com/projectcontour/contour/test/e2e"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func testPathConditionMatch(namespace string) {
```

Comment:
could we use exported test functions maybe instead of duplicating tests, so we can just call them from different suites?

Reply:
Do you know a way to import across test packages?

Reply:
I think our e2e test packages aren't `_test` named so could work like normal?

Reply:
I don't think the `_test.go` files are importable.

Reply:
ah, the filenames could change as well, or we could pull out a helper package maybe

Reply:
If we wanted to get this in as-is, I'd be willing to take the follow-up to do some de-duplication here.